presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -45,6 +46,11 @@ public HiveSessionProperties(HiveClientConfig config)
     {
         sessionProperties = ImmutableList.of(
                 booleanSessionProperty(
+                        BUCKET_EXECUTION_ENABLED,
+                        "Use bucket-aware execution: only use a single worker per bucket",
```

Comment:
Replace "Use" with "Enable" (so we're not repeating that word): "Enable bucket-aware execution: only use a single worker per bucket"

Same change for the config description